vmdb/Gemfile

```diff
@@ -46,7 +46,7 @@ gem 'secure_headers'
 gem 'mime-types'
 # Needed by the REST API
 gem "jbuilder",                       "~>2.0.7"
-gem "gettext_i18n_rails",             "~>1.2.0"
+gem "gettext_i18n_rails",             ">=1.2.3"
```

Comment:
I think you should keep `~>`.  The `>=` could accidentally introduce breaking changes if, say, 2.0 is released with backwards incompatibilities a la [semver](http://semver.org/)